UDPServer/IServerInterface.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UDPServer
+{
+    interface IServerInterface
+    {
+        string ConnectionIP
+        {
+            get;
+            set;
+        }
```

Comment:
The correct formatting is `string ConnectionIP { get; set; }`

Reply:
Changed for both interface and class.